src/main/ast/type.c

```diff
@@ -241,7 +241,51 @@ bool typeImplicitlyConvertable(Type const *from, Type const *to) {
     // float     | -yes-- | n | ------yes------ | no | ---yes---- | --no--
     // double    | -yes-- | n | ------yes------ | no | -----yes------ | n
     // bool      | ------------------------no------------------------ | y
-    return false;  // TODO
+    switch (to->kind) {
```

Comment:
```suggestion
    switch (to->data.keyword.keyword) {
```

I think you want the keyword of the keyword type. (We have a namespace collision between the TypeKind enum and the TypeKeyword enum.)

This change probably applies to every instance of `->kind` you have.

Reply:
I just pushed a commit that I believe fixes this